<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFaultTable</class>
 <widget class="QDialog" name="CFaultTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>故障库</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTableWidget" name="listFault">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;宋体&quot;;</string>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>故障ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>故障名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>用户名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>创建时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>备注</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>厂站名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备名</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="buttonFaultData">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>故障数据</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonFaultAdd">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>添加故障</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonFaultDel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>删除故障</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonFaultModify">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>修改故障</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonStnDev">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>选择厂站设备</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonRefresh">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>刷  新</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonExit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>退  出</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonFaultData</tabstop>
  <tabstop>buttonFaultAdd</tabstop>
  <tabstop>buttonFaultDel</tabstop>
  <tabstop>buttonFaultModify</tabstop>
  <tabstop>buttonRefresh</tabstop>
  <tabstop>buttonExit</tabstop>
  <tabstop>listFault</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonExit</sender>
   <signal>clicked()</signal>
   <receiver>CFaultTable</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>705</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
